src/AppBundle/TonMacron/TonMacronSerializer.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace AppBundle\TonMacron;
+
+use AppBundle\Entity\TonMacronChoice;
+use AppBundle\Entity\TonMacronFriendInvitation;
+
+class TonMacronSerializer
+{
+    /**
+     * @param TonMacronChoice[] $choices
+     *
+     * @return string
+     */
+    public function serializeChoices(array $choices): string
+    {
+        if (!is_iterable($choices)) {
+            throw new \InvalidArgumentException();
+        }
+
+        $handle = fopen('php://memory', 'rb+');
```

Comment:
There is a [CsvEncoder](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Serializer/Encoder/CsvEncoder.php) since symfony 3.2 ;).